test/integration/react-streaming-and-server-components/app/pages/index.server.js

```diff
@@ -27,3 +27,9 @@ export function getServerSideProps({ req }) {
     },
   }
 }
+
+export const config = {
+  amp: false,
+  unstable_runtimeJS: false,
```

Comment:
What's this option for?

Reply:
This is the existing option, I added it for test case that config has multiple properties (I only parsed the 1st one in previous commit)